.github/CODEOWNERS

```diff
@@ -44,8 +44,7 @@
 
 
 # Android team
-# TODO(#11477): Replace @rt4914 with @BenHenning after 2021-01-04.
```

Comment:
Are we expecting this PR to close #11477?

Reply:
I think it can if all of the comments have been addressed.

Reply:
Thanks for the reminder @DubeySandeep! I added "Fix #11477" to the title.